src/atomicui/organisms/SearchBox/SearchBox.tsx

```diff
@@ -48,6 +48,7 @@ const SearchBox: React.FC<SearchBoxProps> = ({
 	const timeoutIdRef = useRef<ReturnType<typeof setTimeout> | null>(null);
 	const [value, setValue] = useState<string>("");
 	const [isFocused, setIsFocused] = useState(false);
+	const [isNLPChecked, setIsNLPChecked] = React.useState(false);
```

Comment:
We used NLP as an abbreviation for Natural Language Processing. I believe you meant to us NL (Natural Language). So `isNLChecked` or `isNLSearchChecked` is more appropriate